web/template/dashboard.gohtml

```diff
@@ -5,10 +5,10 @@
 {{ define "donorCard" }}
   <div class="app-dashboard-card">
     <div class="app-dashboard-row">
-      <h3 class="govuk-heading-m">
+      {{ if .Item.Lpa.RegisteredAt.IsZero }}<h3 class="govuk-heading-m">{{ else }}<h4 class="govuk-heading-m">{{ end }}
```

Comment:
how I wish XHTML 2.0 had won so we could just do `<h>`

Reply:
😂 I am constantly parsing hierarchy on pages like this - nightmare!